uploadNeuroDB/NeuroDB/FileDecompress.pm

```diff
@@ -8,9 +36,19 @@ use Path::Class;
 use Archive::Extract;
 use Archive::Zip;
 
-################################################################
-#####################Constructor ###############################
-################################################################
+
+=pod
+
+=head3 new($file_path) (constructor)
```

Comment:
a minor comment; if we are keeping (constructor); can we make it consistent everywhere?
in the one I just merged, I believe we added the `>>` before the (constructor)...